http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -442,6 +442,9 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
                                         namespaceName
       )
 
+      // The app is blocked in updating now
```

Comment:
What do you mean by blocked? This is just the new spot where the app status transitions from running to updating right?

Reply:
Ah, maybe just a nit to match the GKE interpreter `// Change app status to updating`

Reply:
I wanted to call attention to the fact that if leo restarts after this line, the app is 'unusable'. I'll make it clearer